src/main/java/randoop/output/JavaFileWriter.java

```diff
@@ -73,6 +106,12 @@ private File getDir(String packageName) {
     return dir;
   }
 
+  /**
+   * Returns the output stream to print to the given file.
+   *
+   * @param file the file to be written to
+   * @return the {@code PrintStream} for writing to the file
+   */
   private static PrintStream createTextOutputStream(File file) {
```

Comment:
`PrintStream` seems inappropriate here.
Per https://docs.oracle.com/javase/7/docs/api/java/io/PrintStream.html:

> The PrintWriter class should be used in situations that require writing characters rather than bytes.